settings/src/components/webauthn/list-keys.js

```diff
@@ -109,8 +112,8 @@ function ConfirmRemoveKey( { keyToRemove, onConfirm, onClose, deleting, error }
 				</div>
 			) : (
 				<div className="wporg-2fa__submit-actions">
-					<Button variant="primary" onClick={ onConfirm }>
-						Remove { keyToRemove.name }
+					<Button variant="primary" isDestructive onClick={ onConfirm }>
```

Comment:
I like the message this treatment conveys. There are other modal buttons which could use the same; disable TOTP and disable security keys

Reply:
I think it's useful for things that aren't reversible, like deleting keys. For disabling, I can see it go both ways since you can re-enable them afterwards. 

But since you can non-destructively turn off keys, but destructively disable the TOTP (because you have to set it up again), maybe a blanket "red" button does makes sense.